app/weeklyPicks/WeeklyPicks.tsx

```diff
@@ -161,8 +164,20 @@ const WeeklyPicks = ({ NFLTeams, currentGameWeek }: Props): JSX.Element => {
       });
 
       setUserPick(currentUserPick[user.id].team);
+
+      //toast notification
```

Comment:
Remove these comments. No need for them. It's straight forward what this is for.